packages/utils/regl-renderer/src/rendering/commands/drawLines/meshShaders.js

```diff
@@ -1,3 +1,29 @@
+const meshVert = `
```

Comment:
it is just one of probably few steps to refactor the shaders. the move is because that code is executed first, so it is more convenient for me to look at it. Also the added newlines are to be able visually compare with vColorShaders.
The difference in shader when mesh is single color versus when it is multicolor is very small, so my aim is to do a variant like threejs or others do, to compose the shader based on the options supported.